pkg/node/volumemgr.go

```diff
@@ -301,7 +301,8 @@ func (m *VolumeManager) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.R
 			// drive must be present in the system
 			drive, _ := m.crHelper.GetDriveCRByVolume(volume)
 			if drive != nil {
-				m.addVolumeStatusAnnotation(drive, volume.Name, apiV1.Removed)
```

Comment:
What was the goal for adding annotation? If I remember correctly the idea is to update drive CR and trigger reconcile loop to do extra work...

Reply:
Removing annotation will trigger reconcilation too